test_bootstrap.php

```diff
@@ -1,3 +1,42 @@
 <?php
 
+exec(__DIR__ . "/tests");
+
+ini_set("memory_limit", "512M");
+
 define("TMP", "/home/vagrant/caketmp/");
```

Comment:
Cake doesn't have a way of configuring the path to the temp directory, so we have to set this constant before it can. We need to change the temp directory to be a folder which is not shared with the host OS, or the permissions-related tests will not pass.